src/baseline_bots/utils.py

```diff
@@ -404,7 +407,9 @@ def get_state_value(bot, game, power_name, option="default"):
                 orders=orders[: min(bot.rollout_n_order, len(orders))],
             )
         game.process()
-    return len(game.get_centers(power_name))
+        if movement_phase >= bot.rollout_length:
+            break
+    return len(game.get_centers(power_name)) + 0.5 * len(game.powers[power_name].units) + 0.3*len(game.get_power(power_name).influence)
```

Comment:
This function was previously returning an `int`, and now it returns a `float`. Can this be handled properly by whatever is calling it?

Reply:
this is called and being sorted by returned value. Yes, It is handled. Thanks for asking!